torch/_torch_docs.py

```diff
@@ -4409,6 +4409,10 @@ def parse_kwargs(desc):
 If :attr:`some` is ``True`` (default), the returned `U` and `V` matrices will
 contain only :math:`min(n, m)` orthonormal columns.
 
+.. note:: The implementation of SVD on CPU uses the LAPACK routine `?gesdd` (a divide-and-conquer
+          algorithm) instead of `?gesvd` for speed. However, the SVD on GPU uses the MAGMA routine
+          `gesvd` since it takes up lesser workspace area.
```

Comment:
nit: `?gesvd`

Reply:
I didn't add the `?` considering it is a MAGMA routine that was being discussed there.

Reply:
I just realized that GPUs were running on `gesdd` instead of `gesvd`